modules/dailymotionBidAdapter.md

```diff
@@ -0,0 +1,73 @@
+# Overview
+
+```
+Module Name: Dailymotion Bid Adapter
```

Comment:
"Dailymotion *Bid* Adapter" or "Dailymotion *Prebid* Adapter" ? 
In the description we have "prebid"

Reply:
Dailymotion Bid Adapter is ok. Other bid adapter also has similar name.